CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 4.3.0
+
+- Implements Client#close method, allows Infinity for `http2SessionIdleTime` parameter
+- Adds missed tests for FQL queries
+- Fixes cross-platform global object
+- Notifies about new package version and shows release notes on install
+- Exposes parseJSON from utils
+- Adds `ngram` to special cases for snake_case to CamelCase conversion
```

Comment:
```suggestion
- Adds `NGram` to special cases for snake_case to CamelCase conversion
```

Reply:
sure, I've fixed it,
thanks a lot for a thorough review 👍